src/ProjectTemplates/Web.ProjectTemplates/content/BlazorWeb-CSharp/BlazorWeb-CSharp/Components/Layout/NavMenu.razor

```diff
@@ -27,5 +31,41 @@
                 <span class="bi bi-list-nested" aria-hidden="true"></span> Weather
             </NavLink>
         </div>
+        @*#if (IndividualLocalAuth)
+
+        <div class="nav-item px-3">
+            <NavLink class="nav-link" href="auth">
+                <span class="bi bi-lock" aria-hidden="true"></span> Auth Required
+            </NavLink>
+        </div>
+
+        <AuthorizeView>
+            <Authorized>
+                <div class="nav-item px-3">
+                    <NavLink class="nav-link" href="/Account/Manage">
+                        <span class="bi bi-person-fill" aria-hidden="true"></span> @context.User.Identity?.Name
+                    </NavLink>
+                </div>
+                <div class="nav-item px-3">
+                    <LogoutForm id="logout-form" />
+                    <NavLink class="nav-link" href="#" onclick="document.getElementById('logout-form').submit(); return false;">
```

Comment:
Don't have `href="#"`. I think you can just have `href` (with no value) but better still would be not having the `href` attribute at all if you're not using it.

BTW I now see how you're submitting the logout form, and this is quite unusual! The more normal solution would be for the logout link to be a button inside the form, styled to look as text like the rest.

However I can imagine it would be quite a disruptive change to make right now. Given the time pressure I'd be totally fine with sticking with your `onclick` solution here.

No advantage in passing in the `id` though - it can just be hardcoded inside `LogoutForm` (as far as I can tell).